### PR TITLE
Repo Cleanup Changes

### PR DESCRIPTION
**Pull request by abhayks1**
- Delete BrandedToken.sol,  Internal.sol.
- Delete openst-protocol directory. Move the followings to contracts/ directory: mock version of EIP20Token and EIP20Interface.
- Remove Workers.sol, OpsManaged.sol, Owned.sol
- Renamed EIP20Interface to EIP20TokenInterface.
- Integrate EIP20TokenInterface.sol with TokenHolder and TokenRules.

### CODE REVIEW

**Comment by abhayks1** [comment]
Fixes #43

**Comment by rachinkapoor** [comment]
@deepesh-kn , @gulshanvasnani & @kedarchandrayan , please review this PR independently so that we can merge it (if it looks good).

File: test/multisigwallet/is_transaction_confirmed.js [hunk 1]
```diff
@@ -31,8 +31,7 @@ contract('MultiSigWallet::isTransactionConfirmed', async (accounts) => {
                 nonExistingTransactionID,
                 { from: helper.wallet(0) },
             ),
-            'Should revert as transaction ID does not exist.',
-            'Transaction does not exist.',
```
Comment (pgev) [inline test/multisigwallet/is_transaction_confirmed.js]: Why this check has been removed?

File: test/token_rules/check_global_constraints.js [hunk 2]
```diff
@@ -43,7 +43,6 @@ contract('TokenRules::checkGlobalConstraints', async () => {
                 ),
                 'Should revert as transfers "to" and "amount" arrays length '
                 + 'are not equal.',
-                '\'to\' and \'amount\' transfer arrays\' lengths are not equal.',
```
Comment (pgev) [inline test/token_rules/check_global_constraints.js]: Why this check has been removed?

File: test/multisigwallet/is_transaction_confirmed.js (hunk 1 above)
Comment (abhayks1) [reply test/multisigwallet/is_transaction_confirmed.js]: running truffle test gave me below error:

 AssertionError [ERR_ASSERTION]: 
The contract should revert with:
	"Transaction does not exist." 
instead received:
	"Returned error: VM Exception while processing transaction: revert"

File: test/token_rules/check_global_constraints.js (hunk 2 above)
Comment (abhayks1) [reply test/token_rules/check_global_constraints.js]: Same error as above:

1) Contract: TokenRules::checkGlobalConstraints
       Contract: Negative Tests
         Reverts if transfersTo and transfersAmount array lengths are not equal.:

      AssertionError [ERR_ASSERTION]: 
The contract should revert with:
	"'to' and 'amount' transfer arrays' lengths are not equal." 
instead received:
	"Returned error: VM Exception while processing transaction: revert"

**Comment by pgev** [comment]
fixes #43

**Pull request merged**